internal/controller/util/misc.go

```diff
@@ -36,7 +36,8 @@ const (
 
 	CreatedByRamenLabel = "ramendr.openshift.io/created-by-ramen"
 
-	VGSCRDName = "volumegroupsnapshots.groupsnapshot.storage.k8s.io"
+	VGSCRDPrivateName = "volumegroupsnapshots.groupsnapshot.storage.openshift.io"
+	VGSCRDName        = "volumegroupsnapshots.groupsnapshot.storage.k8s.io"
```

Comment:
If the k8s version of the CRD is installed, we will claim to support CG with Volsync, but for example in `CreateOrUpdateVolumeGroupSnapshot` we would create the private G(roup)V(ersion)K(ind) API. So I am uncertain how this helps?

The older code was checking for the k8s version, whereas we support the private version, so I understand the fix required (i.e check for the private version instead).